ui/src/App.vue

```diff
@@ -34,26 +34,59 @@
     </v-app-bar>
 
     <v-content>
+      <work-space
+        :individuals="savedIndividuals"
+        @clearSpace="savedIndividuals = []"
+        @draggedIndividual="addSavedIndividual"
+      />
       <v-container max-width="900">
-        <individuals-table :fetch-page="getIndividualsPage" />
+        <individuals-table
+          :fetch-page="getIndividualsPage"
+          @saveIndividual="addSavedIndividual"
+        />
       </v-container>
+      <v-snackbar v-model="snackbar">
+        Individual already in work space
+      </v-snackbar>
     </v-content>
   </v-app>
 </template>
 
 <script>
-import { getPaginatedIndividuals } from "./apollo/queries";
+import { getPaginatedIndividuals, getProfileByUuid } from "./apollo/queries";
 import IndividualsTable from "./components/IndividualsTable";
+import WorkSpace from "./components/WorkSpace";
 
 export default {
   name: "App",
   components: {
-    IndividualsTable
+    IndividualsTable,
+    WorkSpace
+  },
+  data() {
+    return {
+      savedIndividuals: [],
+      snackbar: false
+    };
   },
   methods: {
     async getIndividualsPage(page, items) {
       const response = await getPaginatedIndividuals(this.$apollo, page, items);
       return response;
+    },
+    async addSavedIndividual(uuid) {
+      const isSaved = this.savedIndividuals.find(individual => {
+        return individual.identities[0].uuid === uuid;
+      });
+      if (isSaved) {
+        this.snackbar = true;
+        return;
+      }
+      const response = await getProfileByUuid(this.$apollo, uuid);
```

Comment:
Is this needed? Can't we reuse the information that is already on the entry? The less we visit the server, the better. Although, if it's too complicated, we can leave it this way.

Reply:
I had to make a few changes and move the logic around a bit for it to work without the request, but it's better not to visit the server a second time for the same info.